server/views/pages/licence/HDC_AP.njk

```diff
@@ -59,30 +59,39 @@
          manage your return into the community. You will also be under the supervision of a nominated officer. The
          objectives of this supervision are to (a) protect the public, (b) prevent re-offending and (c) help you to
          resettle successfully into the community.</li>
-      <li>Your Home Detention Curfew commences on <span class="entry">{{ licence.homeDetentionCurfewApprovedDate | datetimeToDate or "MISSING - HDC Approved Date"}}</span> and expires on <span class="entry">{{ licence.actualReleaseDate | datetimeToDate or licence.conditionalReleaseDate | datetimeToDate or "MISSING - Actual Release Date"}}</span></li>
-      <li>Your supervision commences on <span class="entry">{{ licence.homeDetentionCurfewApprovedDate | datetimeToDate or "MISSING - HDC Approved Date"}}</span> and expires on <span class="entry">{{ licence.licenceExpiryDate | datetimeToDate or "MISSING - Licence expiry date" }}</span> unless this licence is previously revoked.</li>
+      <li>Your Home Detention Curfew commences on <span class="entry">{{ licence.actualReleaseDate | datetimeToDate or "MISSING - HDC Approved Date/Actual Release Date"}}</span> and expires on <span class="entry">{{ licence.conditionalReleaseDate | datetimeToDate or "MISSING - Conditional Release Date"}}</span></li>
+      <li>Your supervision commences on <span class="entry">{{ licence.actualReleaseDate | datetimeToDate or "MISSING - HDC Approved Date/Actual Release Date"}}</span> and expires on <span class="entry">{{ licence.licenceExpiryDate | datetimeToDate or "MISSING - Licence Expiry Date" }}</span> unless this licence is previously revoked.</li>
```

Comment:
Is the start date of both of these meant to be ARD?

Reply:
Thanks Seb, that's a question we've added to a questions doc to double check and discuss at a later date 😅 so tbc but will be checked.